src/jit/codegenxarch.cpp

```diff
@@ -6070,7 +6070,8 @@ void CodeGen::genJmpMethod(GenTree* jmp)
             //
 
             // Update varDsc->lvArgReg and lvOtherArgReg life and GC Info to indicate varDsc stack slot is dead and
-            // argReg is going live. Note that we cannot modify varDsc->GetRegNum() and lvOtherArgReg here because another
+            // argReg is going live. Note that we cannot modify varDsc->GetRegNum() and lvOtherArgReg here because
```

Comment:
```
Could you please reformat it not to have single words on a new lines?
You could do it manually or merge the whole comment in one line and run `jit-format --c CORE_CLR --fix`

// argReg is going live. Note that we cannot modify varDsc->GetRegNum() and lvOtherArgReg here 
// because another basic block may not be expecting it. 
// Therefore manually update life of argReg.  Note that GT_JMP marks

```

Reply:
I tried jit-format, it made too many changes. I ended up taking your version.